### PR TITLE
UI

### PR DESCRIPTION
**Pull request by SoorajModi**
#37 
- Improves home page
- Improves interface to add Botomir to guild
- Added donation button
- Added 404 page
- Added documentation page

### CODE REVIEW

File: views/404.handlebars [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<main role="main">
+    <div class="container center-content">
+        <h1>404 Not Found</h1>
+    </div>
+</main>
```
Comment (MarshallAsch) [inline views/404.handlebars:5]: add trailing newline.

File: views/docs.handlebars [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<main role="main">
+    <div class="container center-content">
+    </div>
+</main>
```
Comment (MarshallAsch) [inline views/docs.handlebars:4]: trailing newline

File: views/home.handlebars [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
+        <h5>Your friendly neighbourhood Discord bot</h5>
+        <div class="container col-md-offset-3 col-md-5">
+            <a class="btn btn-custom btn-block"
+               href="https://discord.com/api/oauth2/authorize?client_id={{clientID}}
+            &permissions=469887088&scope=bot" role="button" target="_blank">Add to Server</a>
+            <a class="btn btn-custom btn-block" href="https://paypal.me/soorajmodi?locale.x=en_US" role="button"
+               target="_blank">Donate</a>
+        </div>
+    </div>
+</main>
```
Comment (MarshallAsch) [inline views/home.handlebars:14]: trailing new line

File: app.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ app.use((req, res, next) => {
     next();
 });
 
-app.use('/assets', express.static('assets'));
+app.use(express.static('static'));
```
Comment (MarshallAsch) [inline app.js:27]: update the `help` embedded message logo url. and delete the old assets folder

File: views/layouts/main.handlebars [hunk 5]
```diff
@@ -1,50 +1,45 @@
 <!DOCTYPE html>
```
Comment (MarshallAsch) [inline views/layouts/main.handlebars:1]: Should we make a favicon version of the botomir icon and add it to this?
We should also get the updated file from Kiara

**Review by MarshallAsch — CHANGES_REQUESTED** [review]
I have a thing for trailing new lines.

File: views/404.handlebars (hunk 1 above)
Comment (SoorajModi) [reply views/404.handlebars]: Will keep this in mind in the future lmao

File: app.js (hunk 4 above)
Comment (SoorajModi) [reply app.js:27]: Updated

File: views/layouts/main.handlebars (hunk 5 above)
Comment (SoorajModi) [reply views/layouts/main.handlebars:1]: Yes, but she seems really busy right now + her surgery, so im going to wait a bit longer before poking her again. But we will definitely be doing that + updating any other images once it is available.

**Review by MarshallAsch — APPROVED** [review]

**Pull request merged**